shared/src/androidMain/kotlin/di/PlatformModules.android.kt

```diff
@@ -7,24 +7,31 @@ import db.database.car.CarDatabase
 import db.database.car.CarDatabaseWrapper
 import db.database.chargehub.ChargeHubDatabase
 import db.database.chargehub.ChargeHubDatabaseWrapper
+import db.database.level.LevelDatabase
+import db.database.level.LevelDatabaseWrapper
 import db.database.user.UserDatabase
 import db.database.user.UserDatabaseWrapper
 import org.koin.dsl.module
 
 actual fun platformModules() = module {
 
     single {
-        val driver = AndroidSqliteDriver(ChargeHubDb.Schema, get(), "chargehubdb.db")
+        val driver = AndroidSqliteDriver(ChargeHubDb.Schema, get(), DATABASE_NAME)
```

Comment:
I saw you changed this to extract the driver, don't forget to implement this everywhere

Reply:
Extracting the driver is only possible for iOS as far as Ive been able to figure out, and thank you